lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -31,14 +31,23 @@
 
                     this.countries = TeamUtils.selectorOptionsArrayToHashWithBlank(this.context.countries);
                     this.languages = TeamUtils.selectorOptionsArrayToHashWithBlank(this.context.languages);
+                    this.topic = options.topic;
 
                     this.listenTo(this.model, 'change', this.render);
                 },
 
                 render: function() {
                     var memberships = this.model.get('membership'),
                         discussionTopicID = this.model.get('discussion_topic_id'),
-                        isMember = TeamUtils.isUserMemberOfTeam(memberships, this.context.userInfo.username);
+                        isMember = TeamUtils.isUserMemberOfTeam(memberships, this.context.userInfo.username),
+                        isInstructorManagedTopic = false,
```

Comment:
Looks like you can declare this variable with the proper conditioned value if you use a private helper function